ee/codegen/src/generators/nodes/bases/base.ts

```diff
@@ -255,23 +257,85 @@ export abstract class BaseNode<
     });
   }
 
+  protected getRetryAdornment(): AdornmentNode | undefined {
+    if (this.nodeData.adornments && this.nodeData.adornments.length > 0) {
+      const retryAdornment = this.nodeData.adornments.find(
+        (adornment) => adornment.base && adornment.base.name === "RetryNode"
+      );
```

Comment:
[blocking] no reason to make this Retry specific - we'll want to support adornments generally here to lower the friction to supporting some of the others we have soon